rero/rero-ils
# Merge "Subscription" branch to dev

## How to test

* `./run-tests.sh`

----

1) [ ] As a system_librarian, create a new patron_type or edit an existing patron_type with a valid subscription amount. 
2) [ ] edit a patron to link it with this patron type.
3) [ ] see in patron "fees" tab, the newly created subscription fee.
4) [ ] create a second new patron_type with a valid subscription.
5) [ ] edit the same patron and link it to this new second patron type.
6) [ ] see in patron "fees" tab, no new subscription will be created because the patron has already a pending subscription.


## Code review check list
